quesma/proxy/l4_proxy.go

```diff
@@ -106,17 +106,18 @@ func configureRouting() *http.ServeMux {
 			return
 		}
 
-		err := ndjson.BulkForEach(func(operation types.BulkOperation, document types.JSON) {
+		err := ndjson.BulkForEach(func(i int, operation types.BulkOperation, _, document types.JSON) error {
```

Comment:
I would suggest to change name of `i` parameter to be more meaningful

Reply:
Change the parameter name from `i` to `entryNumber`